server/entitymanager/entitymanager.go

```diff
@@ -219,6 +270,11 @@ func parseSecurityArtifacts(artifactDir string) (*service.SecurityArtifacts, err
 	}, nil
 }
 
+// ssBase64 check if a string is base64 encoded.
+func isBase64(str string) bool {
```

Comment:
I might be wrong but I thought we agreed that we wouldn't base64 encode the OV if it's passed as bytes. Therefore doesn't that make this check unneeded?

Reply:
yes, here I am checking if  user provided ov is base64 (in inventory file), then we decode and send it as binary to the router.

Reply:
I see, thanks.